parse_emails/handle_eml.py

```diff
@@ -457,8 +457,14 @@ def extract_address_eml(eml, entry):
 
 def get_attachment_filename(part):
     attachment_file_name = None
-    if part.get_filename():
-        attachment_file_name = str(make_header(decode_header(part.get_filename())))
+    filename = part.get_filename()
+    if filename:
+        try:
+            attachment_file_name = str(make_header(decode_header(filename)))
+        except LookupError:
+            if 'windows-874' in filename:
```

Comment:
add comment explaining that files that are encoded in windows-874 have that string in the file name